<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupbox_buttons">
      <property name="minimumSize">
       <size>
        <width>1002</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit_statusBar">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="minimumSize">
          <size>
           <width>630</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>630</width>
           <height>80</height>
          </size>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QPushButton" name="btn_next">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>30</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Next (N)</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_back">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Back (B)</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_save">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>30</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Save (S)</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_deleteSelected">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>30</y>
             <width>111</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete Selected (D)</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_deleteUnsaved">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>30</y>
             <width>91</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete Unsaved</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_deleteAll">
           <property name="geometry">
            <rect>
             <x>530</x>
             <y>30</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete All</string>
           </property>
          </widget>
          <zorder>btn_next</zorder>
          <zorder>btn_back</zorder>
          <zorder>btn_save</zorder>
          <zorder>btn_deleteSelected</zorder>
          <zorder>btn_deleteUnsaved</zorder>
          <zorder>btn_deleteAll</zorder>
          <zorder>textEdit_statusBar</zorder>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="img_area">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>1002</width>
        <height>587</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>214</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="tableWidget_categories">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>207</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideLeft</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerItem</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="btn_applyCategory">
               <property name="text">
                <string>Apply Category (E)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_chooseColor">
               <property name="text">
                <string>Choose Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_addCategory">
               <property name="text">
                <string>Add Category</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_deleteCategory">
               <property name="text">
                <string>Delete Category</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_refresh">
               <property name="text">
                <string>Refresh</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_saveCategory">
               <property name="text">
                <string>Save to file</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="setting_area">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>230</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label_startFrame">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Start Frame</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_frameStep">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Frame Step</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_totalFrames">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Total Frames</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_totalFrames_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>120</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>N.A.</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_currentFrame">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Current Frame</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_currentFrame_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>150</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>N.A.</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_applySetting">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>190</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit_startFrame">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="plainText">
           <string>N.A.</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit_frameStep">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>50</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="plainText">
           <string>N.A.</string>
          </property>
         </widget>
         <zorder>label_startFrame</zorder>
         <zorder>label_frameStep</zorder>
         <zorder>label_totalFrames</zorder>
         <zorder>label_totalFrames_2</zorder>
         <zorder>label_currentFrame</zorder>
         <zorder>label_currentFrame_2</zorder>
         <zorder>btn_applySetting</zorder>
         <zorder>plainTextEdit_startFrame</zorder>
         <zorder>plainTextEdit_frameStep</zorder>
         <zorder>stackedWidget_2</zorder>
         <zorder>groupBox_3</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLabeling_for_Yolo">
    <property name="geometry">
     <rect>
      <x>-1621</x>
      <y>100</y>
      <width>176</width>
      <height>94</height>
     </rect>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_openfile"/>
    <addaction name="act_opendir"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_openUsage"/>
   </widget>
   <addaction name="menuLabeling_for_Yolo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_openfile">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
  <action name="act_opendir">
   <property name="text">
    <string>Open Image Directory</string>
   </property>
  </action>
  <action name="actions">
   <property name="text">
    <string>s</string>
   </property>
  </action>
  <action name="act_openUsage">
   <property name="text">
    <string>Open Usage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
